x-pack/plugins/fleet/common/types/models/epm.ts

```diff
@@ -36,7 +36,7 @@ export type EpmPackageInstallStatus = 'installed' | 'installing';
 export type DetailViewPanelName = 'overview' | 'policies' | 'settings' | 'custom';
 export type ServiceName = 'kibana' | 'elasticsearch';
 export type AgentAssetType = typeof agentAssetTypes;
-export type AssetType = KibanaAssetType | ElasticsearchAssetType | ValueOf<AgentAssetType>;
+export type AssetType = KibanaAssetType | ElasticsearchAssetType | ValueOf<AgentAssetType> | 'rules';
```

Comment:
You'll probably have to get someone from the `epm` team to also give you a LGTM, but looking below I think they might want it like so (from their patterns):

```ts
export type AssetType =
  | KibanaAssetType
  | ElasticsearchAssetType
  | ValueOf<AgentAssetType>
  | RuleAssetType;

/*
 Enum mapping of a rules saved object asset type
*/
export enum RuleAssetType {
  rules = 'rules',
}
```

(caveat, I haven't tested that, just coming off the top of my head)